R/controls.R

```diff
@@ -30,7 +30,7 @@ jqueryLib <- function() {
 }
 
 #' @export
-filter_select <- function(id, label, sharedData, group, allLevels = FALSE,
+filter_select <- function(id, label, sharedData, group = "default", allLevels = FALSE,
   multiple = TRUE) {
```

Comment:
@jcheng5 do we even need a group argument here? It could just derive from the `sharedData`'s group property, right?